de.metas.swat/de.metas.swat.base/src/test/java/de/metas/invoicecandidate/api/impl/InvoiceCandBLHandlCompleteAndReversalTest.java

```diff
@@ -313,6 +322,79 @@ private void doTestCreditMemoReinvoicableAndReversal(final boolean reverseInvoic
 		assertThat(invoiceCandBL.sumupQtyInvoicedAndNetAmtInvoiced(ic).get().getLeft().getStockQty().toBigDecimal(), comparesEqualTo(expectedQtyInvoicedSumAfterReversal));
 	}
 
+	private void doTestCreditMemoReversal( final BigDecimal reversalIlaExpectedQtyInvoiced)
```

Comment:
pls add a javadoc with a brief description of what the new test does. not required before integrating, but it would be good to add it soon